src/clj_kondo/impl/namespace.clj

```diff
@@ -93,6 +93,18 @@
                    :else (recur raw-ns
                                 (next ns-list))))))))))
 
+(defn lint-unknown-clauses
+  [ctx clauses]
+  (doseq [clause clauses]
+    (findings/reg-finding!
+     ctx
+     (node->line (:filename ctx)
+                 clause
+                 :unknown-ns-option
+                 (str "Unknown ns option: " (if (sequential? clause)
```

Comment:
Can you give an example when the sequential check here is important? Is this illustrated by the tests?

Reply:
The intention was to only show the relevant part of the line, but it doesn't actually work because the `clause` is a rewrite-clj node, not a clojure sexp. I'll remove this.